src/elm/overloaded.js

```diff
@@ -74,8 +74,10 @@ class Except extends Expression {
 
   exec(ctx) {
     const [a, b] = this.execArgs(ctx);
-    if (a == null || b == null) {
+    if (a == null && b == null) {
       return null;
+    } else if (b == null) {
+      return a;
```

Comment:
This behavior is actually specific only to lists.  From the spec on [List#Except](http://build.fhir.org/ig/HL7/cql/09-b-cqlreference.html#except-1):
> If the second argument is null, the operation is performed as though the second argument was an empty list.

BUT... from the spec on [Interval#except](http://build.fhir.org/ig/HL7/cql/09-b-cqlreference.html#except):
> If either argument is null, the result is null.

So you need to branch based on `typeIsArray(a)` to decide how a null `b` should be handled.